crates/starknet-server/src/api/json_rpc/mod.rs

```diff
@@ -259,8 +291,40 @@ impl std::fmt::Display for StarknetRequest {
     }
 }
 
+#[derive(Serialize, Deserialize, Debug)]
+#[serde(untagged)]
+pub(crate) enum StarknetResponse {
+    BlockWithTransactionHashes(Block),
+    BlockWithFullTransactions(Block),
+    StateUpdate(StateUpdate),
+    StorageAt(Felt),
+    TransactionByHash(Transaction),
+    TransactionByBlockAndIndex(Transaction),
+    TransactionReceiptByTransactionHash(Box<TransactionReceipt>),
```

Comment:
Why Box?

Reply:
clippy complaint about it being very large compared to other structs

Reply:
cool from clippy